src/Form/NewsroomSettingsForm.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_newsroom\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\oe_newsroom\OeNewsroom;
+
+/**
+ * Newsroom Settings Form.
+ */
+class NewsroomSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      OeNewsroom::CONFIG_NAME,
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'newsroom_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config(OeNewsroom::CONFIG_NAME);
+
+    $form['universe'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Universe Acronym'),
```

Comment:
No capitalisation of words.
```suggestion
      '#title' => $this->t('Universe acronym'),
```